### PR TITLE
total expected efforts for fixed budget

### PR DESCRIPTION
**Pull request by Abhinegi2**
Targets #2239

### Description
 1- if project has Fixed Budget selected, we need to use that project Total Estimated Hours and project Duration.
 2-For Fixed Budget project type Total monthly expected efforts for project = Total Estimated Hours/Duration.
 3-After that we need to calculate Daily Expected Hours and Weekly Expected Hours for that project.
 4-For Monthly billling , No calculation,as it is.Total will be the sum of Individual expected hours.

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by gautam071** [comment]
@Abhinegi2 Please fix the CI checks

**Comment by gautam071** [comment]
@Abhinegi2 I am able to see @Himanshunegi0332 commits in your PR as well
Can you check that once?

**Comment by Abhinegi2** [comment]
> @Abhinegi2 I am able to see @Himanshunegi0332 commits in your PR as well Can you check that once?

i checked, this is happened because i forget about changing  my git config email and username .

File: Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php [hunk 1]
```diff
@@ -27,18 +27,40 @@
...
-                        (@{{ totalDailyEffort*5 }} H)
+                    ({{  $dailyEffort*5 }} H)
                     </div>
+                    @endif
+                    @if( $project->type === 'monthly-billing')
                     <div class="col-1">
                         Monthly
-                        (@{{ totalDailyEffort*workingDaysInMonth }} H)
+                    (@{{ totalDailyEffort*workingDaysInMonth }} H)
                     </div>
```
Comment (gautam071) [inline Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php:56]: Looks like some code are repeating can we make the repeated code command like `Daily` `Weekly` etc.

**Review by gautam071 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**